packages/design-system/src/components/OcSelect/OcSelect.vue

```diff
@@ -183,19 +265,42 @@ export default defineComponent({
   color: var(--oc-color-input-text-default);
 
   .vs {
-    &__search,
-    &__selected {
+    &__search {
       color: var(--oc-color-input-text-default);
+      // padding: 0.625em;
     }
+
     &__selected {
+      color: var(--oc-color-input-text-default);
+      // padding: 0.25em 0.55em;
       background-color: var(--oc-color-background-default);
     }
     &__search::placeholder,
     &__dropdown-toggle,
     &__dropdown-menu {
-      background-color: var(--oc-color-input-bg);
+      -webkit-appearance: none;
+      background-clip: padding-box, border-box;
+      background-color: var(--oc-color-background-highlight);
+      background-origin: border-box;
+      border-radius: 0;
+      border-radius: 6px;
```

Comment:
Would `5px` also be fine? We have a bit of a chaotic usage of border-radius values, `5`, `15` and `30` pixels seem to be the most common values.